framework/src/base/EigenProblem.C

```diff
@@ -154,11 +151,10 @@ EigenProblem::solve()
   Moose::perf_log.pop("Eigen_solve()", "Execution");
 }
 
-
-#if LIBMESH_HAVE_SLEPC
 bool
 EigenProblem::converged()
 {
+#if LIBMESH_HAVE_SLEPC
   return _nl_eigen->converged();
+#endif  
```

Comment:
This will throw a compiler error for people without SLEPc - an empty body of a function that should return `bool`.